packages/datetime/src/dateRangeInput.tsx

```diff
@@ -148,6 +196,23 @@ export class DateRangeInput extends AbstractComponent<IDateRangeInputProps, IDat
         this.setState({ isOpen: false });
     }
 
+    private getInitialRange = (props = this.props): moment.Moment[] => {
```

Comment:
nit: Maybe just call it `getInitialMomentRange` instead of declaring a return type?